### DISCUSSION TITLE
[Help]: Can someone help me with an example of Fullscreen like the one we have for Flickity

### DISCUSSION THREAD

**Discussion by Pixsal** [davidjerleke/embla-carousel#985] [Help] [answered]
### Summary

https://flickity.metafizzy.co/options#fullscreen

this is what i am trying to achieve using vanilla js but with Embla Carousel. A bit of help would be greatly appreciated.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by sarussss** [comment]
Hi @Pixsal 
https://pkprcq.csb.app/

- https://github.com/davidjerleke/embla-carousel/discussions/702#discussioncomment-8514242.

**Reply by Pixsal** [reply]
Hi @sarussss Thanks for the reply, But i am not well versed with React, Is there a vanilla js version for it?

**Reply by davidjerleke** [reply]
@Pixsal in that case, this is a duplicate of:

- #702

…and a full screen example on the documentation website is on the roadmap:

- #841 

A contribution like a PR or [sponsoring](https://github.com/sponsors/davidjerleke) this project might speed up the process of adding the example to the docs.

**Reply by sarussss** [reply, marked as answer]
HI @Pixsal 
Try this example, hope it helps you.
https://t4wldy.csb.app/
https://codesandbox.io/p/sandbox/embla-carousel-generator-vanilla-forked-t4wldy?file=%2Fsrc%2Fjs%2Findex.js%3A43%2C42

**Reply by Pixsal** [reply]
Thanks a bunch @sarusss this helps greatly.
